docs-web/configuration/policy-resource.md

```diff
@@ -140,6 +138,8 @@ policies:
 
 ### RateLimit
 
+> **Feature Status**: Rate-Limiting is available as a preview feature: it is suitable for experimenting and testing; however, it must be used with caution in production environments. Additionally, while the feature is in preview status, we might introduce some backward-incompatible changes to the resource specification in the next releases.
```

Comment:
Better to include the information about how to enable the preview policies (here and for the other preview policies)

**Feature Status**: Rate-Limiting is available as a preview feature: it is suitable for experimenting and testing; however, it must be used with caution in production environments. Additionally, while the feature is in preview status, we might introduce some backward-incompatible changes to the resource specification in the next releases. The feature is disabled by default. To enable it, set the [`enable-preview-policies`]() command-line argument of the Ingress Controller.